test/exchangeIssuance/exchangeIssuanceLeveragedForCompound.spec.ts

```diff
@@ -0,0 +1,1122 @@
+import "module-alias/register";
```

Comment:
As mentioned on discord we also need integration tests in addition to these unit tests.